components/scream/src/share/grid/point_grid.cpp

```diff
@@ -1,4 +1,6 @@
 #include "share/grid/point_grid.hpp"
+#include "ekat/kokkos/ekat_kokkos_utils.hpp"
+#include "../../physics/share/physics_constants.hpp"
```

Comment:
I would avoid relative paths when including. You can use the path relative to the `src` folder: 

```
#include "physics/share/physics_constants.hpp"
```

Reply:
At some point we should move the constants to `src/share`, cause it's a bit ugly when the share folder includes stuff in the physics one. Yes, it's only headers, so it does not create target dependencies, but still..